config/site_settings.yml

```diff
@@ -2338,6 +2338,10 @@ developer:
     default: ""
     client: true
     hidden: true
+  experimental_topic_bulk_actions:
```

Comment:
Sorry but I think this needs to be a group setting; it works as a boolean in this first PR, but when we get the other modals online people are going to want to test this, and we don't want to be too disruptive.

As we've had issues with other client-level group-based settings (see /t/119198 internally) we need to also serialize this true/false for the user like so:

```ruby
def use_experimental_topic_bulk_actions?
  scope.user.in_any_groups?(SiteSetting.experimental_topic_bulk_actions_enabled_groups_map)
end
```

Reply:
ohhhh didn't realize this was a thing, I see now.